City-of-Pori/pori-kada
# Feature/pori 112 place element

To test:

1. drush fra -y && drush cc all
2. Go to /asuminen-ja-ympäristö/asuminen/asukastoiminta and make sure that if you add pages to "Linkki" under the "Ei käytössä" tab they are not displayed on the node page.
3. Edit the page and add some "Related service offering".
4. You should now see a "Places" block after the content. Make sure it looks nice and works.
5. Make sure that the plus and minus icons on the site (main navigation on desktop and mobile, footer navigation on mobile) look good. They have been refactored to use font instead of separate icon. Remember however that there might be one pixel inconsistencies between browsers since they are font icons.
